specification/resources/resource-manager/Microsoft.Authorization/2017-06-01-preview/examples/createOrUpdatePolicySetDefinition.json

```diff
@@ -0,0 +1,44 @@
+{
+  "parameters": {
```

Comment:
these example files need to be referenced in the swagger spec for them to show up in documentation, right?   See https://github.com/pilor/azure-rest-api-specs/blob/current/documentation/x-ms-examples.md

Reply:
Of course they should! thanks for catching this